synapse/lib/types.py

```diff
@@ -442,7 +440,7 @@ def repr(self, valu, defval=None):
         if hit:
             return tuple(vals)
 
-        return defval
+        return str(valu)
```

Comment:
we should never get here since repr() should always return something now

Reply:
Agreed.  Needs cleanup.